bootstrap/src/main/resources/minimal-genesis.block.json

```diff
@@ -0,0 +1,84 @@
+{
+"blockV1": {
+  "payload": {
+    "transactions": [
+      {
+        "payload": {
+          "reducedPayload": {
+            "commands": [
+              {
+                "addPeer": {
+                  "peer": {
+                    "address": "localhost:10001",
+                    "peerKey": "d04da271b57fe63426ae1dc97f6952104037411fcf4f3b739dc217f45e5fc99b"
+                  }
+                }
+              },
+              {
+                "createRole": {
+                  "roleName": "tester",
```

Comment:
MB change tester role name to more meaningful? For example, `seed`.

Reply:
updated